src/ddocs/ddocs.rst

```diff
@@ -32,19 +32,47 @@ them extensively within each function:
 - :ref:`Database Security object <security_object>`
 - :ref:`Guide to JavaScript Query Server <query-server/js>`
 
-Creation
-========
+Creation and structure
```

Comment:
Minor nit: maybe use a capital `S` in `Structure` just for consistency with other titles. I don't actually remember the correct rule there but consistency would be better I think.

Reply:
Well spotted! I just pushed the fix. Thank you.